### PR TITLE
Merge template 2.9

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Merge template 2.9

### CODE REVIEW

File: assets/multiqc_config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/proteinfold" target="_blank">nf-core/proteinfold</a>
+  This report has been generated by the <a href="https://github.com/nf-core/proteinfold/1.1.0dev" target="_blank">nf-core/proteinfold</a>
```
Comment (maxulysse) [inline assets/multiqc_config.yml:2]:
```suggestion
  This report has been generated by the <a href="https://github.com/nf-core/proteinfold" target="_blank">nf-core/proteinfold</a>
```

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/proteinfold" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/proteinfold/1.1.0dev/output" target="_blank">documentation</a>.
```
Comment (maxulysse) [inline assets/multiqc_config.yml:4]:
```suggestion
  <a href="https://nf-co.re/proteinfold" target="_blank">documentation</a>.
```

**Review by maxulysse — CHANGES_REQUESTED** [review]
almost perfect template merge

**Comment by JoseEspinosa** [comment]
Will be great if you could review it @maxulysse

**Comment by maxulysse** [comment]
> Will be great if you could review it @maxulysse

I reviewed and asked for changes but you marked them as resolved and didn't anything about it

**Comment by JoseEspinosa** [comment]
Your changes were applied here [3a2e413](https://github.com/nf-core/proteinfold/pull/111/commits/3a2e4134e803cb47000f3888decb3495d6e06792) and here [ba11c1c](https://github.com/nf-core/proteinfold/pull/111/commits/ba11c1c43e49a74dc313bfd1765d43ea761de5ae). But the thing is not that then the file reverts to how it was currently in dev https://github.com/nf-core/proteinfold/blob/dev/assets/multiqc_config.yml ?

**Review by maxulysse — APPROVED** [review]
LGTM

**Pull request merged**